fitbenchmarking/core/results_output.py

```diff
@@ -208,14 +208,17 @@ def colour_highlight(value):
         table.index = html_links
         table_style = table.style.applymap(colour_highlight)\
             .set_caption(title)
-        root = os.path.dirname(os.path.abspath(__file__))
-        style_css = os.path.join(root, 'HTML_templates/style_sheet.css')
-
-        style = '<link rel="stylesheet" type="text/css"  ' \
-            'href="{0}" />'.format(style_css)
-        with open(name + 'html', "w") as f:
-            f.write(style)
-            f.write(table_style.render(table_styles=style_css))
+        root = os.path.dirname(inspect.getfile(fitbenchmarking))
+        html_page_dir = os.path.join(root, 'HTML_templates')
+        style_css = os.path.join(html_page_dir, 'style_sheet.css')
+        env = Environment(loader=FileSystemLoader(html_page_dir))
+        template = env.get_template("blank_page.html")
+
+        output_file = os.path.join(name + 'html')
```

Comment:
This is a no-op. I think join just puts a '/' in between each of it's args (or on other os' could put a different symbol in).
`name + 'html'` should be fine.